### ISSUE TITLE
Installing and requiring an npm module results in build failure

### ISSUE THREAD

**Issue by Lokeh** [thheller/shadow-cljs#739]
Reproduction steps:

1. Create shadow-cljs project with a basic `:browser` build
2. Start build watch
3. Try to require an npm module in a namespace, e.g. `"react-dom"`. Observe error informing one may need to install it.
4. install dependency

Expected behavior:
The build completes successfully.

Actual behavior:
The build fails with error:
```
[:app] Build failure:
no output for id: [:shadow.build.classpath/resource "goog/base.js"]
{:resource-id [:shadow.build.classpath/resource "goog/base.js"]}
ExceptionInfo: no output for id: [:shadow.build.classpath/resource "goog/base.js"]
```

Restarting the shadow-cljs process resolves the error.

**Comment by orestis** [comment]
I’m seeing similar errors. Doing a force compile (from the UI) fixes things, I recall.

**Comment by thheller** [comment]
FWIW I have also seen this error pop up sometimes but I cannot explain it. I can also not reproduce it with the steps above.

**Comment by thheller** [comment]
Nevermind. Found it. Fixed in `2.10.13`.
